zulip_botserver/zulip_botserver/server.py

```diff
@@ -28,6 +29,39 @@ def read_config_section(parser: configparser.ConfigParser, section: str) -> Dict
     }
     return section_info
 
+def read_config_from_env_vars(bot_name: Optional[str] = None) -> Dict[str, Dict[str, str]]:
+    bots_config = {}  # type: Dict[str, Dict[str, str]]
+    env_config = OrderedDict()   # type: Dict[str, Dict[str, str]]
+
+    # Check for presence of environment variables
+    # otherwise throws appropriate errors
+    if os.environ.get('ZULIP_BOTSERVER_CONFIG') is None:
+        raise OSError("Could not read environment varibale 'ZULIP_BOTS': Variable not set.")
+
+    # The bots emails keys sites and tokens
+    # are stored in their respective lists
+    env_config = json.loads(str(os.environ.get('ZULIP_BOTSERVER_CONFIG')), object_pairs_hook=OrderedDict)
```

Comment:
Do we need the `str` here?

Reply:
Yes because lint throws an error, as an environment variable can be None: resulting in type mismatch.
Though I have a check for None before, lint does not understand that.

Reply:
If you pull this out to its own variable, and use it in the `if` as well as here, then mypy will propagate that `is None` constraint.  It'll repeat the name of the environment variable less (for fewer potential copy/paste mistakes in the future), and also have fewer arbitrary variable casts!